sdk/servicebus/service-bus/test/backupMessageSettlement.spec.ts

```diff
@@ -45,117 +48,60 @@ describe("Backup message settlement - Through ManagementLink", () => {
+  });
 
-  describe("Operations On Messages After Receiver Is Closed", function(): void {
-    afterEach(async () => {
-      await afterEachTest();
-    });
+  async function sendReceiveMsg(testMessages: ServiceBusMessage): Promise<ReceivedMessageWithLock> {
+    await sender.sendMessages(testMessages);
+    const msgs = await receiver.receiveMessages(1);
 
-    async function sendReceiveMsg(
-      testMessages: ServiceBusMessage
-    ): Promise<ReceivedMessageWithLock> {
-      await sender.sendMessages(testMessages);
-      const msgs = await receiver.receiveMessages(1);
-
-      should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
-      should.equal(msgs.length, 1, "Unexpected number of messages");
-      should.equal(msgs[0].body, testMessages.body, "MessageBody is different than expected");
-      should.equal(
-        msgs[0].messageId,
-        testMessages.messageId,
-        "MessageId is different than expected"
-      );
-      should.equal(msgs[0].deliveryCount, 0, "DeliveryCount is different than expected");
-
-      return msgs[0];
-    }
-
-    describe("Complete a message", function(): void {
-      async function testComplete(): Promise<void> {
-        const testMessages = entityNames.usesSessions
-          ? TestMessage.getSessionSample()
-          : TestMessage.getSample();
-        const msg = await sendReceiveMsg(testMessages);
-        await receiver.close();
-        let errorWasThrown = false;
-        try {
-          await msg.complete();
-        } catch (err) {
-          should.equal(
-            err.message,
-            `Failed to ${DispositionType.complete} the message as the AMQP link with which the message was received is no longer alive.`,
-            "Unexpected error thrown"
-          );
-          errorWasThrown = true;
-        }
+    should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
+    should.equal(msgs.length, 1, "Unexpected number of messages");
+    should.equal(msgs[0].body, testMessages.body, "MessageBody is different than expected");
+    should.equal(msgs[0].messageId, testMessages.messageId, "MessageId is different than expected");
+    should.equal(msgs[0].deliveryCount, 0, "DeliveryCount is different than expected");
 
-        receiver = await serviceBusClient.test.getPeekLockReceiver(entityNames);
-        if (entityNames.usesSessions) {
-          should.equal(errorWasThrown, true, "Error was not thrown for messages with session-id");
-          const msgBatch = await receiver.receiveMessages(1);
-          await msgBatch[0].complete();
-        } else {
-          should.equal(errorWasThrown, false, "Error was thrown for sessions without session-id");
-        }
-        await testPeekMsgsLength(receiver, 0);
+    return msgs[0];
+  }
+
+  [noSessionTestClientType, withSessionTestClientType].forEach((testClientType) => {
+    it(testClientType + ": complete() removes message", async function(): Promise<void> {
```

Comment:
How about bringing all the `it` blocks here under one `describe` block and wrapping this 
`[noSessionTestClientType, withSessionTestClientType].forEach((testClientType) => {` 
around that `describe` block so that we don't have to repeat this for every test?

Reply:
That is a good idea!

Done with https://github.com/Azure/azure-sdk-for-js/pull/9950/commits/66682d5abe7e3d15762ce2a3a2cb0ca49b9a31fd

This is how the results would look like

<img width="536" alt="Screen Shot 2020-07-08 at 12 45 16 PM" src="https://user-images.githubusercontent.com/16890566/86963140-e6932400-c118-11ea-87d0-8ac63de86abc.png">